core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -75,7 +75,8 @@ case class RawlsBillingProjectResponse(projectName: RawlsBillingProjectName,
                                        roles: Set[ProjectRoles.ProjectRole],
                                        status: CreationStatuses.CreationStatus,
                                        message: Option[String],
-                                       managedAppCoordinates: Option[AzureManagedAppCoordinates]
+                                       managedAppCoordinates: Option[AzureManagedAppCoordinates], // TODO: do we even want to return this?
+                                       cloudPlatform: String
```

Comment:
My first thought was to make this of type `bio.terra.profile.model.CloudPlatform` (which is an enum), but I had trouble with the JSON serialization format declaration for that.